phpstan.neon

```diff
@@ -0,0 +1,7 @@
+includes:
+    - ./vendor/larastan/larastan/extension.neon
+
+parameters:
+    level: 5
+    paths:
+        - src
```

Comment:
This is missing its end of line. Note the big red circle.

Things like this must be caught during review.